docs/website/contents/for-developers/OperationalCertificateIssueNumber.md

```diff
@@ -0,0 +1,81 @@
+
+The benefit of identities including an OCIN Y rather than merely being determined by a ColdKey X is that an SPO can increment their OCIN whenever they suspect the adversary could have acquired their current HotKey (eg they discover evidence of unauthorized software running on their block-producing node).
+This response will be effective because the header validity and tiebreaker rules involve the OCIN.
+
+- The protocol state maintains a mapping from X to Z, the OCIN of the youngest header issued by X on this chain.
+  The protocol rules allow exactly two identities to issue a header extending some chain when X is elected, either Y=Z or Y=Z+1.
+- The Cardano tiebreaker rules favor the greater OCIN when comparing two headers from X in the same slot.
+  Without this preference in the tiebreaker, whether X is able to increment their OCIN on the honest chain is merely a network race between the adversary's header/block and the SPO's header/block.
+  Conservatively, the adversary is assumed to have better network connectivity, and so theoretically could prevent an honest party from incrementing the OCIN until the leaked HotKey expires up to 90 days later (see next section).
+
+The fact that Z is not allowed to decrease along a chain prevents the adversary that acquired a HotKey with precedence Y from extending alternative chains whenever X is elected unless that chain branches off from the honest chain _before_ X incremented their OCIN to Y+1 on the honest chain---see the visualization below.
+An SPO is thereby able to effectively revoke the rights from a HotKey that the adversary might have acquired, thereby re-establishing control of their blocks' predecessor and payload on the honest chain.
+
+```mermaid
+flowchart TD
+    I["Z(X)=0"] --> X1[...] --> X2["Z(X)=1"] --> X3[...]
+    I           --> Y1["(X,0) ok"]
+    X1          --> Z1["(X,0) ok"]
+    X2          --> A1["(X,0) invalid!"]
+```
+
+The fact that Z can only increase along a chain by at most one per header bounds the rate at which X can increment their Z as a chain grows.
+If that rate weren't bounded, an adversary X could generate infinite identities Y whenever they wanted, and the Cardano tiebreaker would favor each new one.
+
+## Advantage over Mere ColdKey-HotKey Indirection
+
+Completely orthogonally to OCINs, each HotKey is confined to one interval of slots, the {KesPeriod}, which is ~90 days on Cardano.
+This achieves _forward-security_ ([Wikipedia](https://en.wikipedia.org/wiki/Forward_secrecy)) with respect to long-term time scales without requiring SPOs to access their ColdKey unbearably often (recall that it's physically offline).
+
+If Cardano did not involve OCINs, then an adversary would benefit from an acquired HotKey until its KesPeriod expired, which could be up to 90 days.
+On the other hand, an SPO---especially one with a lot of delegated stake---can increment the OCIN much sooner than 90 days after an attack (assuming they notice it).
```

Comment:
Maybe worth to mention that there are potential alternatives, like submitting the OCIN update via a tx, or (conceptually similar) via a stake pool re-registration certificate (would need to be enriched). This is also how eg Peras voting (and I presume Leios voting) will handle this scenario.

These approaches generally have longer latency to come into effect, but work even if the adversary is quite strong and can orphan blocks that the corrupted pool is producing (they just need one nearby election).

Reply:
Ah, that was a bad omission on my part, thanks for catching it! It severely trims the potential 90 day period down to 36 hours + tx latency for the cost of a small tx fee---right?

> would need to be enriched

I haven't though through it, but it seems like a minor change. Have you already considered some details? I'll ask Andre.

Reply:
Ah, OK. I was mentally thinking that the old cold key signing the new cold key would let us circumvent some of these delays. But that's skipping some practical considerations---eg "opcerts" would be chains of signatures, which is no good. I suppose I was just loosely envisioning a Frankenstein of the existing opcert issue number mechanism.

I'm now thinking that I should have interpreted "enriched" your original text as "Letting a tx (monoidally) update the opcert issue number map after a forecast window of delay".